### PR TITLE
Linter Error corrections for Contrast Task

### PR DESCRIPTION
**Pull request by dbrothersKenna**
Misc corrections per Linter errors.
https://github.com/KennaSecurity/toolkit/runs/2822866084?check_suite_focus=true

### CODE REVIEW

**Comment by dbrothersKenna** [comment]
@davidaustinarcher
Cleaned up some linting errors in the Contrast code. Please review if you can to ensure that didn't change what you were doing.

Thanks,
Dbro

File: tasks/connectors/contrast/lib/client.rb [hunk 1]
```diff
@@ -115,14 +117,13 @@ def get_trace_recommendation(id, rule_name)
         end
 
         def get_trace_story(id)
-          begin
-            url = "#{@base_url}/traces/#{id}/story"
+          # begin
```
[the remark this answers is not in the document]
Comment (shotop) [reply tasks/connectors/contrast/lib/client.rb:120]: nvm - TIL the method definition line can act as the begin in a rescue block

**Review by shotop — APPROVED** [review]
The only outstanding question I have is whether or not we want to keep all of the old commented code in the changed files here.  Otherwise I think it's good to go.

File: tasks/connectors/contrast/contrast.rb [hunk 2]
```diff
@@ -293,60 +286,73 @@ def run(opts)
...
+          "HIGH" => 8,
+          "MEDIUM" => 6,
+          "LOW" => 4,
+          "UNIMPORTANT" => 2
+        }
+        importance_lookup[importance]
+        # case importance
+        # when "CRITICAL"
+        #   "10"
+        # when "HIGH"
```
[the remark this answers is not in the document]
Comment (shotop) [reply tasks/connectors/contrast/contrast.rb:324]: Same here - I was just confused by old commented code still in the file after rubocop updates.

File: tasks/connectors/contrast/lib/client.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (dbrothersKenna) [reply tasks/connectors/contrast/lib/client.rb:120]: yep. Thanks. Original author had the rescue block but Rubocop considered it redundant.

File: tasks/connectors/contrast/contrast.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (dbrothersKenna) [reply tasks/connectors/contrast/contrast.rb:324]: Just me being old fashioned.  I didn't want to remove until I was sure the new version functioned.

**Pull request merged**